python/ray/serve/tests/test_http_state.py

```diff
@@ -624,15 +630,15 @@ def check_health_obj_ref_not_none():
 
 
 @patch("ray.serve._private.http_state.PROXY_HEALTH_CHECK_PERIOD_S", 0.1)
-def test_update_draining(mock_get_all_node_ids, setup_controller, all_nodes):
+@pytest.mark.parametrize("number_of_worker_nodes", [1])
```

Comment:
probably want to test >1 as well, might be a weird edge case

Reply:
this is the number of worker, so there are 2 nodes total with the head node. But yea, I can add a case with 2 and 3 worker nodes :)